mobile/examples/whisper/android/azure/app/src/main/java/ai/onnxruntime/example/speechrecognition/SpeechRecognizer.kt

```diff
@@ -0,0 +1,86 @@
+package ai.onnxruntime.example.speechrecognition
+
+import ai.onnxruntime.OnnxTensor
+import ai.onnxruntime.OrtEnvironment
+import ai.onnxruntime.OrtSession
+import ai.onnxruntime.extensions.OrtxPackage
+import android.os.SystemClock
+
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import java.lang.Exception
+
+@Serializable
+data class Error(
+    val message: String,
+    val type: String,
+    val code: String
+)
+
+@Serializable
+data class Response(
+    val text: String,
+)
+
+@Serializable
+data class ErrorResponse(
+    val error: Error
+)
+
+class SpeechRecognizer(modelBytes: ByteArray) : AutoCloseable {
+    private val session: OrtSession
+    private val baseInputs: Map<String, OnnxTensor>
+
+    init {
+        val env = OrtEnvironment.getEnvironment()
+        val sessionOptions = OrtSession.SessionOptions()
+        sessionOptions.registerCustomOpLibrary(OrtxPackage.getLibraryPath())
+
+        session = env.createSession(modelBytes, sessionOptions)
+
+        // NOTE!! Modify the following line and input your own OpenAI AUTH_TOKEN for successful API request calls.
+        // By default, it will return an error message (given no correct authToken provided.)
+        // DO NOT hardcode an auth token here. See `User Settings` section in your OpenAI account for more detailed information
+        // to find your API Key.
```

Comment:
nit: how someone wants to manage their auth token is up to them so I wouldn't say 'DO NOT'. We could maybe suggest they store it in https://developer.android.com/reference/androidx/security/crypto/EncryptedSharedPreferences.